integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java

```diff
@@ -99,122 +68,49 @@ public void close() throws Exception {
-
-    }
-
-    private void cleanupEndpoints() throws RemoteException, EndpointAdminEndpointAdminException {
-        EndpointTestUtils.cleanupDefaultEndpoint(ENDPOINT_NAME, endPointAdminClient);
-    }
-
-    private void endpointAdditionScenario() throws Exception {
-        int beforeCount = endPointAdminClient.getEndpointCount();
-
-        addEndpoint(AXIOMUtil.stringToOM("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-                + "<endpoint xmlns=\"http://ws.apache.org/ns/synapse\" name=\"" + ENDPOINT_NAME + "\">\n"
-                + "    <loadbalance algorithm=\"org.apache.synapse.endpoints.algorithms.RoundRobin\">\n"
-                + "        <endpoint name=\"endpoint_urn_uuid_7E71CCD625D839E55A25548428059450-1123062013\">\n"
-                + "            <address uri=\"http://webservices.amazon.com/AWSECommerceService/UK/AWSECommerceService.wsdl\"/>\n"
-                + "        </endpoint>\n" + "    </loadbalance>\n" + "</endpoint>"));
-
-        int afterCount = endPointAdminClient.getEndpointCount();
-        assertEquals(1, afterCount - beforeCount);
-
-        String[] endpoints = endPointAdminClient.getEndpointNames();
-        if (endpoints != null && endpoints.length > 0 && endpoints[0] != null) {
-            List endpointList = Arrays.asList(endpoints);
-            assertTrue(endpointList.contains(ENDPOINT_NAME));
-        } else {
-            fail("Endpoint has not been added to the system properly");
-        }
-    }
-
-    private void endpointStatisticsScenario() {
-        try {
-            endPointAdminClient.enableEndpointStatistics(ENDPOINT_NAME);
-        } catch (EndpointAdminEndpointAdminException e) {
-            return;
-        } catch (RemoteException e) {
-            return;
-        }
-        fail("Enabling statistics on a load-balance endpoint did not cause an error");
-    }
-
-    private void endpointDeletionScenario() throws RemoteException, EndpointAdminEndpointAdminException {
-        int beforeCount = endPointAdminClient.getEndpointCount();
-        endPointAdminClient.deleteEndpoint(ENDPOINT_NAME);
-        EndpointTestUtils.assertDefaultEndpointDeletion(beforeCount, endPointAdminClient);
-
-    }
-
-    private void uploadResourcesToConfigRegistry() throws Exception {
-
-        resourceAdminServiceClient
-                .addCollection("/_system/config/", "test_ep_config", "", "Contains test Default EP files");
-        resourceAdminServiceClient
-                .addResource("/_system/config/test_ep_config/loadbalancingEP_Test.xml", "application/xml", "xml files",
-                        new DataHandler(new URL("file:///" + getClass().getResource(
-                                "/artifacts/ESB/endpoint/loadbalancingEndpointConfig/loadbalancingEP_Test.xml")
-                                .getPath())));
-        Thread.sleep(1000);
+        Assert.assertTrue(response.contains("Response from server: Server_3"));
```

Comment:
Shall we add assert messages?

Reply:
Done in https://github.com/wso2/micro-integrator/pull/574